### PR TITLE
fix the english grammar

### PR DESCRIPTION
**Pull request by brianteeman**
This simple PR fixes the grammar in a description committed in 13830

This will not affect translations as presumably they didnt write with bad grammar in their own languages

### CODE REVIEW

**Comment by infograf768** [comment]
> This will not affect translations as presumably they didnt write with bad grammar in their own languages

Changes like this always affect translators using crowdin and com_localise advanced as they appear as a change that has to be approved.

OK for the change. Just a reminder.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 9c90026295c5ff96ff3f35908bf0c62f4bbe70f3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14449">issues.joomla.org/tracker/joomla-cms/14449</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 9c90026295c5ff96ff3f35908bf0c62f4bbe70f3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14449">issues.joomla.org/tracker/joomla-cms/14449</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14449">issues.joomla.org/tracker/joomla-cms/14449</a>.</sub>

File: administrator/language/en-GB/en-GB.com_menus.ini [hunk 1]
```diff
@@ -85,7 +85,9 @@ COM_MENUS_ITEM_FIELD_ASSIGNED_LABEL=" Menu"
 COM_MENUS_ITEM_FIELD_ASSOCIATION_NO_VALUE="- No association -"
 COM_MENUS_ITEM_FIELD_BROWSERNAV_DESC="Target browser window when the menu item is selected."
 COM_MENUS_ITEM_FIELD_BROWSERNAV_LABEL="Target Window"
-COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Select the menu items that should be shown or not under this container. If there are no items to show, then this container will also be hidden.<br>Please note that when you install a new component it will be displayed by default until you come back here and hide it too."
+COM_MENUS_ITEM_FIELD_CLIENT_ID_DESC="Choose the client application menu"
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_menus.ini]: I'm confused why do we add two language strings here? Without any use. We have removed them here: https://github.com/joomla/joomla-cms/pull/14140

**Review by zero-24 — CHANGES_REQUESTED** [review]
As per comment. ;)

**Comment by brianteeman** [comment]
This pr was written a long time ago

**Comment by brianteeman** [comment]
@zero-24 see #14460

**Pull request closed without merging**